server/config/config.go

```diff
@@ -306,6 +308,8 @@ func (man Manager) addConfigs() {
 		"(DEPRECATED: Use filesystem.result_log_file) Path for osqueryd result logs")
 	man.addConfigBool("osquery.enable_log_rotation", false,
 		"(DEPRECATED: Use filesystem.enable_log_rotation) Enable automatic rotation for osquery log files")
+	man.addConfigDuration("osquery.max_jitter_label_update_interval", 0,
```

Comment:
1) Would this perhaps be easier to set as a percent?
2) Should it be nonzero by default? Osquery for example uses a default 10% skew on schedule intervals.